common/SC_PaUtils.hpp

```diff
@@ -0,0 +1,167 @@
+// helper functions for SuperCollider server's PortAudio backend
+
+#ifndef SC_PAUTILS_HPP
+#define SC_PAUTILS_HPP
+
+#include "portaudio.h"
+#include <stdlib.h>
+
+enum class IOType { Input, Output };
+
+static std::string GetDeviceName(int index) {
```

Comment:
this function should also have a more specific name since it is now in the global namespace. I really wish we had some sort of namespacing in this project, but that is not your problem :)